Code/SkinModHelperConfig.cs

```diff
@@ -1,21 +1,48 @@
 ﻿using Microsoft.Xna.Framework;
 using System.Collections.Generic;
+using Monocle;
+using YamlDotNet.Serialization;
 
 namespace Celeste.Mod.SkinModHelper {
-    public class SkinModHelperConfig {
-        public string SkinId { get; set; }
-        public string SkinDialogKey { get; set; }
-        public List<HairColor> HairColors { get; set; }
-
-        public class HairColor {
-            public int Dashes { get; set; }
-            public string Color { get; set; }
-        }
-
-        public List<Color> GeneratedHairColors { get; set; }
-
-        public string GetUniquePath() {
-            return SkinId.Replace('_', '/') + '/';
-        }
+    public class SkinModHelperConfig {
+        public string Options { get; set; }
+        public bool Player_List { get; set; }

```

Comment:
It's generally bad form to use both CamelCase and underscores in the same identifier. Stick to CamelCase for C# public class variables.